.github/workflows/build_tests.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
     - name: Wait for workload completion and confirm it succeeded
       run: python3 xpk.py workload list --cluster $TPU_CLUSTER_NAME --zone=us-central2-b --wait-for-job-completion $WORKLOAD_NAME --timeout 300
     - name: Run a Pathways workload on Ubuntu base image
-      run: python xpk.py workload create-pathways --cluster $TPU_CLUSTER_NAME --workload $PATHWAYS_WORKLOAD_NAME --docker-image='marketplace.gcr.io/google/ubuntu2004' --tpu-type=v4-8 --num-slices=2 --zone=us-central2-b --command "bash test.sh" 
```

Comment:
Hmm not following why we are not using the test.sh script?

Reply:
ah since it is using a --docker-image approach not the ---base-docker-image approach. Surprised actually this test worked in the past interesting!

Reply:
I think earlier this test was not failing with user workload failure, which a good confirmation that the current fix works.

Reply:
Ah I see, thanks for helping me understand!

Could you also add the flag to other workloads that are created in the test? I think those will face the same problem potentially?